app/controllers/admin/items_in_maintenance_controller.rb

```diff
@@ -0,0 +1,8 @@
+module Admin
+    class ItemsInMaintenanceController < BaseController
```

Comment:
We're starting to move the report controllers into the `Admin::Reports` namespace, and it would be great to put this one there as well.